helper_test.go

```diff
@@ -972,3 +972,28 @@ func randomString(t *testing.T) string {
 	}
 	return v
 }
+
+// skips a test if ENABLE_TFE is not set to equal "1"
+// this is used to conditionally skip tests if not running against a Terraform
+// Enterprise instance. Effectively these tests will only run against a Terraform
+// Enterprise instance.
+func skipIfNotEnterprise(t *testing.T) {
```

Comment:
Might be a bit clearer as `skipIfCloud`; Although the mechanism is "enterprise or not", the semantics of the skip names can be cloud vs. enterprise for clarity.

Reply:
Agreed, the wording is off here, will fix